resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/DecommissionSuite.scala

```diff
@@ -54,9 +54,84 @@ private[spark] trait DecommissionSuite { k8sSuite: KubernetesSuite =>
       executorPatience = None,
       decommissioningTest = true)
   }
+
+  test("Test basic decommissioning with shuffle cleanup", k8sTestTag) {
+    sparkAppConf
+      .set(config.DECOMMISSION_ENABLED.key, "true")
+      .set("spark.kubernetes.pyspark.pythonVersion", "3")
```

Comment:
We can remove this because the default is `3` now and there is no `2` in Apache Spark 3.1.

Reply:
Sounds good :)